.prettierignore

```diff
@@ -7,4 +7,5 @@ js/private/node-patches/fs.js
 min/
 npm/private/test/vendored/
 js/private/worker/worker.js
-js/private/worker/src/worker_protocol.ts
\ No newline at end of file
+js/private/worker/src/worker_protocol.ts
+js/private/js_image_layer.mjs
```

Comment:
could you add it to .gitattributes as well so later changes to it don't appear in the code review?
same with coverage.js which appears edited in this PR